$MirOS: ports/games/frozen-bubble/patches/patch-server_game_c,v 1.2 2006/11/04 04:12:04 tg Exp $
--- server/game.c.orig	Sun Nov 12 20:47:14 2006
+++ server/game.c	Sat Dec 23 01:36:55 2006
@@ -38,6 +38,10 @@
 #include "log.h"
 #include "game.h"
 
+#ifndef MSG_NOSIGNAL
+#define MSG_NOSIGNAL 0
+#endif
+
 enum game_status { GAME_STATUS_OPEN, GAME_STATUS_CLOSED, GAME_STATUS_PLAYING };
 
 #define MAX_PLAYERS_PER_GAME 5
